content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,208 @@ you can view processes in other containers.
 
 * Learn how to [debug pods using ephemeral containers](/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container).
 
+### Kubernetes Ephemeral Container Security
+Attempting to debug a Pod and realizing that you can't install curl due to security settings has to be a meme at this point. Good security practices are always nice but it often comes at the cost of usability. To the point where some may even solve this problem by installing debug tools into their production images. Shudders.
+
+Kubernetes has introduced a new concept called ephemeral containers to deal with this problem. Ephemeral containers are temporary containers that can be attached after a Pod has been created. Rejoice! We can now attach a temporary container with all the tools which we desire. While the applications container may have "annoying security features" like a read only file system the ephemeral container can enjoy all the freedom which writing files entails. I love this feature so I need to upgrade my cluster immediately
+
+
+## Digging Deeper
+
+Now that we have the new feature we can start a ephemeral container in any Pod we like  
+
+```shell
+kubectl run ephemeral-demo --image=k8s.gcr.io/pause:3.1 --restart=Never
+kubectl debug -it ephemeral-demo --image=busybox:1.28
+```
+
+We get a shell and life is now much simpler, but wait a minute. This post is not about how to use ephemeral containers, there are enough of those already, but rather the security implications of enabling ephemeral containers. Let's have a look at the YAML for the Pod that we created the ephemeral container in.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ephemeral-demo
+spec:
+  ...
+  ephemeralContainers:
+  - name: debugger-r59b7
+    image: busybox:1.28
+    imagePullPolicy: IfNotPresent
+    stdin: true
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    tty: true
+```
+                            
+Interesting, there is a new field called ephemeralContainers in the Pod definition. This new field contains a list of containers similar to initContainers and containers. It is not identical as there are certain options which are not available, refer to the API documentation for more information. It does however allow configuration of the container security context, which could in theory allow a bad actor to escalate the container's privileges. This should not affect those of us who use a policy enforcement tool right? The answer is yes and no depending on the tool and version that is being used. It also depends on if you are using policies from the project's library or policies developed in house.
+
```

Comment:
Please save the irrelevant statements such as "interesting".
The statement about "new" is warranted to decay pretty soon.
Please try use backticks for field names for readability.

```
The `ephemeralContainers` field in the spec of a Pod ...
```

Reply:
"This should not affect those of us who use a policy enforcement tool right? The answer is yes and no depending on the tool and version that is being used."

Please just state the facts in plain English.